README.md

```diff
@@ -17,4 +17,4 @@ The `App` should contain the next `Sum` components:
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_sum/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Artem193.github.io/react_sum/) and add it to the PR description.
```

Comment:
You should add this link to the PR description. You can click an edit button to change the description 
![image](https://github.com/mate-academy/react_sum/assets/99403198/c41ecce6-08af-4fd4-8884-f9aa113c344b)